crates/bevy_pbr/src/render/morph.rs

```diff
@@ -28,16 +28,16 @@ impl Default for MorphUniform {
 }
 
 pub fn prepare_morphs(
-    device: Res<RenderDevice>,
-    queue: Res<RenderQueue>,
+    render_device: Res<RenderDevice>,
+    render_queue: Res<RenderQueue>,
```

Comment:
i recall there was a PR some time ago to unify the render_/gpu_ prefixes, was that merged and is this conforming to that or anything at all?

Reply:
I prefer shorter names, but _everywhere_ in the engine it uses `render_foo`, so I preferred make it more consistent, especially wrt the new `skin` module. I remember the `gpu_foo` PR, wish it was merged now. :D